website/docs/d/event_trigger.html.markdown

```diff
@@ -64,6 +64,7 @@ In addition to all arguments above, the following attributes are exported:
 * `config_match` - A [$match](https://docs.mongodb.com/manual/reference/operator/aggregation/match/) expression document that MongoDB Realm includes in the underlying change stream pipeline for the trigger. 
 * `config_project` - A [$project](https://docs.mongodb.com/manual/reference/operator/aggregation/project/) expression document that Realm uses to filter the fields that appear in change event objects. 
 * `config_full_document` - If true, indicates that `UPDATE` change events should include the most current [majority-committed](https://docs.mongodb.com/manual/reference/read-concern-majority/) version of the modified document in the fullDocument field.
+* `unordered` - Sort order for `DATABASE` type.
```

Comment:
Sorry I didn't get to you on this but the docs have a better explanation: from https://www.mongodb.com/docs/atlas/app-services/admin/api/v3/#operation/adminCreateTrigger   
```suggestion
* `unordered` - Only Available for Database Triggers. If true, event ordering is disabled and this trigger can process events in parallel. If false, event ordering is enabled and the trigger executes serially.
```